pages/api/content/glamorous.md

```diff
@@ -106,6 +106,64 @@ const MyStyledComponent = glamorous(MyComponent, {
 
 > Try this out in your browser [here](https://codesandbox.io/s/GZEo8jOyy)!
 
+### filterProps
+
+Sometimes it's not possible to know which DOM element will be rendered at the end
+and therefore no `rootEl` can be set to avoid [Unknown Prop Warnings](https://facebook.github.io/react/warnings/unknown-prop.html).
+To filter out these properties from being forwarded to the child components you
+can set `filterProps`. `filterProps` takes an arrays of prop names.
+
+```js
+const withFlex = Component => {
+  return glamorous(Component, {
+    filterProps: ['flex'],
+  })(
+    ({flex}) => (flex ? {display: 'flex'} : undefined),
+  )
+}
+
+const MyComponent = withFlex(props => (<div {...props} />))
+
+<MyComponent flex />
+// this will render a <div> element with the display CSS property set to 'flex'.
+// The withFlex HOC can't set a `rootEl` because it accepts all kind of components.
+// The flex property will not be forwarded to the div because it is listed in
+// `filterProps`.
+```
+
+Setting a property in `filterProps` always takes precedence over `forwardProps`.
+If the same property is contained in both arrays it will always be filtered.
```

Comment:
Honestly, I'm not sure how valuable this bit is. Seems weird for anyone to even try 🤷‍♂️

Reply:
You are right. Nobody would do that by intention. I'll remove that paragraph.